src/screens/SendTokens.js

```diff
@@ -382,17 +397,24 @@ class SendTokens extends React.Component {
     if (hathorLib.wallet.isHardwareWallet()) {
       if (!version.isLedgerCustomTokenAllowed()) {
         // Custom token not allowed for this Ledger version
-        $('#notSupported').modal('show');
+        this.context.showModal(MODAL_TYPES.ALERT_NOT_SUPPORTED, {
+          children: (
+            <div>
+              <p>{t`Unfortunately this feature is not supported with the Hathor app version on your Ledger device. If you need this feature, you can use it by installing the most recent Hathor app.`}</p>
+            </div>
+          )
+        })
         return;
       }
       if (this.state.txTokens.filter(t => !hathorLib.tokens.isHathorToken(t.uid)).length === LEDGER_TX_CUSTOM_TOKEN_LIMIT) {
         // limit is 10 custom tokens per tx
         const modalBody = <p>{t`Ledger has a limit of ${LEDGER_TX_CUSTOM_TOKEN_LIMIT} different tokens per transaction.`}</p>
-        this.setState({
-          ledgerAlertModalTitle: t`Token limit reached`,
-          ledgerAlertModalBody: modalBody,
+        this.context.showModal(MODAL_TYPES.ALERT, {
+          id: 'ledgerAlertModal',
+          title: t`Token limit reached`,
+          body: modalBody,
+          buttonName: 'Ok',
```

Comment:
The same for the `buttonName`

Reply:
4155a44519e9e2a4de667e7564f976f7753f02af fixes it as well